### ISSUE TITLE
Stupid question

### ISSUE THREAD

**Issue by Redas17** [Ling-APE/ComfyUI-All-in-One-FluxDev-Workflow#6]
I am new to this, and I don't understand how I supposed to switch between text to image and image to image? There is no information about it in description, thanks!

**Comment by arthurwolf** [comment]
(I'm super new to this so I might be wrong, I haven't been able to get it to work after hours of trying, but...)

In the middle, there is a "output switch" node, where you can select 1, 2 or 3.

I **think** that:

* `1` is the default sampler based on the "official" workflow (text to image)
* `2` is the custom stuff with the negative prompt (text to image also)
* `3` is the image-to-image.

This is the best I could figure out following from node to node.

**Comment by zhch12121** [comment]
> (I'm super new to this so I might be wrong, I haven't been able to get it to work after hours of trying, but...)
> 
> In the middle, there is a "output switch" node, where you can select 1, 2 or 3.
> 
> I **think** that:
> 
> * `1` is the default sampler based on the "official" workflow (text to image)
> * `2` is the custom stuff with the negative prompt (text to image also)
> * `3` is the image-to-image.
> 
> This is the best I could figure out following from node to node.

that's not true. the real switch for text to image and image to image is in panel "img to img or size picking" > "image source switch". 1 is I2I (image to image), 2 is T2I (text to image)

**Comment by arthurwolf** [comment]
@zhch12121 Thanks for the correction. What's `3` then?

**Comment by zhch12121** [comment]
The third input source in the switch is empty so I guess it's reserved for further input. Should be no use for now.

**Comment by dancingmadkefka** [comment]
I also don't understand how to use it. IMG to IMG seems to be setup for upscaling instead of generating new images with prompts right?

**Comment by doysalim0-dot** [comment]
> I am new to this, and I don't understand how I supposed to switch between text to image and image to image? There is no information about it in description, thanks!

### ****
